benchalerts/benchalerts/pipeline_steps/conbench.py

```diff
@@ -0,0 +1,106 @@
+"""Pipeline steps to talk to Conbench."""
+
+from typing import Any, Dict, Optional
+
+from benchclients.conbench import ConbenchClient
+from benchclients.logging import fatal_and_log, log
+
+from ..alert_pipeline import AlertPipelineStep
+from ..conbench_dataclasses import FullComparisonInfo, RunComparisonInfo
+
+
+class GetConbenchZComparisonStep(AlertPipelineStep):
+    """An ``AlertPipeline`` step to get information from Conbench comparing the runs on
+    a contender commit to their baselines, using a z-score threshold. This is always the
+    first step of the pipeline.
+
+    Parameters
+    ----------
+    contender_sha
+        The commit SHA of the contender commit to compare. Needs to match EXACTLY what
+        Conbench has stored; typically 40 characters. It can't be a shortened version of
+        the SHA.
+    z_score_threshold
+        The (positive) z-score threshold to send to the Conbench compare endpoint.
+        Benchmarks with a z-score more extreme than this threshold will be marked as
+        regressions or improvements in the result. Default is to use whatever Conbench
+        uses for default (at the time of writing, this is 5).
+    conbench_client
+        A ConbenchClient instance. If not given, one will be created using the standard
+        environment variables.
+    step_name
+        The name for this step. If not given, will default to this class's name.
+
```

Comment:
I forgot to move over the environment variable docs. Will do in the next commit.